subprojects/docs/src/docs/release/notes.md

```diff
@@ -343,6 +349,28 @@ The following Attributes have disambiguation rules defined.
 For more information, see [Attribute Disambiguation Rules](userguide/variant_attributes.html#sec:abm_disambiguation_rules).
 
 
+
+<a name="configuration"></a>
+### Configuration Cache
```

Comment:
Piotr: I'd add an introductory paragraph about the configuration cache similarly to previous release notes.